.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "(gdb) Launch",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/build/signalserver",
+            // -dbg Verbose debug messages
+            // -nothreads Turn off threaded processing. It may be useful for debugging, but in line 133 of loss.cc there is a segmentation fault. Another way is to launch the command '-exec set scheduler-locking on' in the debug console
```

Comment:
Change this comment if the segmentation fault has already been fixed.

Reply:
DONE